Classes/Views/PBGitRevisionCell.m

```diff
@@ -336,20 +326,22 @@ - (void) drawWithFrame:(NSRect)rect inView:(NSView *)view
 			[self drawCircleInRect: ownRect];
 	}
 
-
 	if ([self.objectValue refs] && [[self.objectValue refs] count])
 		[self drawRefsInRect:&rect];
-
-	// Still use this superclass because of hilighting differences
-	//_contents = [self.objectValue subject];
-	//[super drawWithFrame:rect inView:view];
-	[textCell setObjectValue: [self.objectValue subject]];
-	[textCell setHighlighted: [self isHighlighted]];
-	[textCell drawWithFrame:rect inView: view];
+	
+	NSRect frame = self.textField.frame;
+	
+	frame.origin.x = rect.origin.x;
+	frame.origin.y = (self.bounds.size.height - frame.size.height) / 2;
+	frame.size.width = self.bounds.size.width - frame.origin.x;
+	
+	self.textField.frame = frame;
```

Comment:
It's a little strange to be calling `setFrame:` on a view inside of `drawRect:`. It might be better to do this in `layout` or `setObjectValue:`. You could split `drawRefsInRect:` into `drawRefsInRect` and `rectForTextField:` so the roles of drawing and calculating view frames are separated.